docs/reference/commandline/plugin_inspect.md

```diff
@@ -12,9 +12,9 @@ parent = "smn_cli"
 # plugin inspect (experimental)
 
 ```markdown
-Usage:  docker plugin inspect PLUGIN
+Usage:  docker plugin [OPTIONS] PLUGIN [PLUGIN...]
```

Comment:
@vdemeester PTAL
Sorry， It is copy-paste mistake.
I have updated this PR.
Thanks!